00-sys-tweaks/files/jacktrip-init.sh

```diff
@@ -168,13 +160,11 @@ function update_device {
 	# Detect sound card
 	detect_hat
 
-	# Keep trying, so that a USB device can later be plugged in
-	while [ "$DEVICETYPE" == "" ]; do
-		echo "Unable to detect sound device"
-		leds_flip
-		sleep 5
-		detect_card
-	done
+	# Use dummy device if no sound card is detected
+	if [ "$DEVICETYPE" == "" ]; then
+		DEVICETYPE="dummy"
+		DEVICENAME="dummy"
```

Comment:
how about `virtual-sound-card` or `digital-sound-card`?

Reply:
These are directly used by jacktrip-agent: https://github.com/jacktrip/jacktrip-agent/blob/develop/cmd/device.go#L289-L307

But I like "virtual" as a replacement for "dummy" when it appears in vs-web